src/vs/workbench/contrib/notebook/browser/view/cellParts/cellComments.ts

```diff
@@ -157,10 +157,12 @@ export class CellComments extends CellContentPart {
 	override didRenderCell(element: ICellViewModel): void {
 		if (element.cellKind === CellKind.Code) {
 			this.currentElement = element as CodeCellViewModel;
-			this.initialize(element);
-			this._bindListeners();
+			this.initializeCodeCell(element);
```

Comment:
should it be `initialize`?

Reply:
> should it be `initialize`?

Thank you for the review. I chose to use initializeCodeCell to explicitly differentiate between the initialization logic for code cells and markup cells, ensuring clear separation and maintainability.
However, to align with your feedback and maintain consistency, I can revert to using the initialize method and ensure it handles both cell types correctly within the method. This approach will streamline the initialization process while still supporting comments for both code and markup cells.